src/app/settings/payment.component.ts

```diff
@@ -8,27 +8,15 @@ import { PaymentMethodType } from 'jslib-common/enums/paymentMethodType';
 
 import { ApiService } from 'jslib-common/abstractions/api.service';
 import { PlatformUtilsService } from 'jslib-common/abstractions/platformUtils.service';
+import { StorageService } from 'jslib-common/abstractions/storage.service';
+import { ConstantsService } from 'jslib-common/services/constants.service';
```

Comment:
This should be in its own import group (i.e. separated by a newline). This is because you're importing a service directly rather than an abstraction.

Reply:
OKay I've added it above the webConstants one so both services are together unless it's better to keep the jslib separated by a new line from this import too?

Reply:
Yes, I think keep it separate from `webConstants` because it's coming from jslib. So the import groups here will look like:
* jslib abstractions
* jslib services (ConstantsService)
* web services (WebConstants)